docker-compose.debug.yaml

```diff
@@ -0,0 +1,60 @@
+# Adds ports to the db and access to the temporal UI for debugging purposes.
+# Expected to be used like this:
+# VERSION=dev docker compose -f docker-compose.yaml -f docker-compose.debug.yaml up
+version: "3.8"
+x-logging: &default-logging
+  options:
+    max-size: "100m"
+    max-file: "5"
+  driver: json-file
+services:
+  db:
+    ports:
+      - 8011:5432
+    networks:
+      - airbyte_internal
+      - airbyte_public
```

Comment:
why are all these airbyte related files here ?
the issue doens't ask you to do anything related to airbyte. please remove.